cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,48 @@
 describe('', () => {
-  before(() => {
+  let user;
+  let article;
 
+  before(() => {
+  });
+  beforeEach(() => {
+    cy.task('generateArticle').then((generatedArticle) => {
+      article = generatedArticle;
+    });
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+    });
   });
 
-  it('', () => {
+  it('should create an article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.createArticle(article.title, article.description, article.body)
+      .as('article');
 
+    // assert article created code
+    cy.get('@article').should((response) => {
+      expect(response.status).to.eq(200);
+    });
+    cy.visit('https://conduit.mate.academy/');
   });
 
-  it('', () => {
+  it('should delete an article', () => {
```

Comment:
This test can be shortened:
![image](https://github.com/mate-academy/qa_cypress_e2e_article_flow/assets/37079719/14a018db-3dc4-497f-be6c-1ff58f1a2170)